ej2-angular/dialog/how-to/close-dialog-while-click-on-outside-of-dialog.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: post
-title: Close dialog while click on outside of dialog in Angular Dialog component | Syncfusion
-description: Learn here all about Close dialog while click on outside of dialog in Syncfusion Angular Dialog component of Syncfusion Essential JS 2 and more.
+title: Close by clicking outside in Angular Dialog Component | Syncfusion
+description: Learn here all about close by clicking outside support in Syncfusion Essential Angular Dialog component, it's elements and more.
```

Comment:
change description and h1 as per meta title.

Reply:
done